dhis-2/dhis-api/src/main/java/org/hisp/dhis/period/RelativePeriodEnum.java

```diff
@@ -57,6 +57,7 @@
     QUARTERS_LAST_YEAR,
     LAST_YEAR,
```

Comment:
Please update the PR title to follow the convention:

https://github.com/dhis2/wow-backend/blob/master/guides/git_commit_messages.md

E.g. "feat: add Last 10 years relative period [DHIS2-7029]"